wsd/FileServer.cpp

```diff
@@ -1017,6 +1019,11 @@ void FileServerRequestHandler::preprocessFile(const HTTPRequest& request,
 
     Poco::replaceInPlace(preprocess, std::string("%USER_INTERFACE_MODE%"), userInterfaceMode);
 
+    std::string useIntegrationThemeString = "true";
+    if (!useIntegrationTheme)
+        useIntegrationThemeString = "false";
```

Comment:
Why not use tertiary operator here too?

`const std::string useIntegrationThemeString = useIntegrationTheme ? "true" : "false";`

Actually, it could even be `constexpr auto`, but simple works too.